src/scraper/tests/el_pitazo_test.py

```diff
@@ -0,0 +1,23 @@
+# TODO move tests out this module when approved.
+
+from scraper.tests.utils import fake_response_from_file
+from scraper.scrapers.el_pitazo_scraper import ElPitazoScraper
+from scraper.settings import ROOT_DIR
+import os 
+
+
+def test_title_parse_ok():
+    url = "tests/html_bodies/el_pitazo_fallas_electricas_carne.html"
+    test_file = os.path.join(ROOT_DIR, url)
+    response = fake_response_from_file(test_file, "https://elpitazo.net/cronicas/la-fallas-electricas-han-disminuido-la-cantidad-de-carne-que-consume-el-venezolano/")
+
+    scraper = ElPitazoScraper()
+    parse_output = scraper.parse(response)
+
+    # body = ????? TODO still don't know how to test body equality
```

Comment:
I think that for the body test, you'll have to copy a super long string and use that 
different ways to make the test a bit nicer to read in this case are:
- define a method in the same test file that returns the expected body as a string, that way you just say
```python
assert parse_output['title'] == getExpectedBody()
 ... 
def getExpectedBody():
      "super long string that you expect to be scraped from the website....." 
```
- define a method as above, but put the expected body in a different text file and read from it 
```python
assert parse_output['title'] == getExpectedBody()
 ... 
def getExpectedBody():
      # <code that will go open a text file and read it as a string>
```

I would suggest the first approach